### DISCUSSION TITLE
New images from cloudinary doesn't load with next/image component

### DISCUSSION THREAD

**Discussion by manowski** [vercel/next.js#25352] [Help]
Hi everyone,

I have some issue with images from cloudinary. When I upload a new one then it doesn't load on the website, the old one works very well. 

To be clear I did something like this:
-uploaded images to cloudinary
-deployed app to server
At this point, everything works well but if I upload new images then I got a 404 error.

My next confing:
```javascript
module.exports = {
    images: {
      domains: [ 'res.cloudinary.com'],
    }
  }
```
Page with images
```javascript
some imports

export const getServerSideProps = async () => {
    const res = await fetch('https://myownapi.com/api/users');
    const data = await res.json();

    return {
        props: { user: data }
    }
}
const UserMainPage = ({ user }) => {
    return (
        <div>
        {user['user_info'].map((user_info, index) => {
            <Image src={user_info.image_url} width={200} height={400} />
        })}
        </div>
    );
}
export default UserMainPage;
```

So like I mentioned when user will have new updated data with new image url and this image will be uploaded to cloudinary, then I can open with browser, on localhost also works but on production server it doesn't load.

I tested with 2 servers:
```javascript
1:
node -v 
v14.17.0

"next": "10.2.2"

2:
node -v 
v14.17.0

"next": "10.1.3"
```

On both there is the same problem

**Comment by le-minh-duc-dev** [comment]
I have the same issues
